plugin/storage/cassandra/schema/create.sh

```diff
@@ -26,11 +26,19 @@ dependencies_ttl=${DEPENDENCIES_TTL:-0}
 # $ cqlsh -e "show version"
 # [cqlsh 5.0.1 | Cassandra 3.11.11 | CQL spec 3.4.4 | Native protocol v4]
 #
-cas_version=$(cqlsh -e "show version" \
-    | awk -F "|" '{print $2}' \
-    | awk -F " " '{print $2}' \
-    | awk -F "." '{print $1}' \
-)
+if [ -z "$PASSWORD" ]; then
+  cas_version=$(cqlsh -e "show version" \
+      | awk -F "|" '{print $2}' \
+      | awk -F " " '{print $2}' \
+      | awk -F "." '{print $1}' \
+  )
+else
+  cas_version=$(cqlsh -u ${USER} -p ${PASSWORD} -e "show version" \
```

Comment:
Is there another way to pass credentials? Passing via cmd line arguments is not secure, they can be observed via ps.

Reply:
I'm afraid not. Seems only `$CQLSH_HOST` and `$CQLSH_PORT` can be provided as env vars, and the authentication can only be performed through CLI args or plain config file ([see ref](https://docs.datastax.com/eol/en/security/6.0/security/secCqlshCredentials.html)).

In fact, there's already calls of this kind on the parent file [`docker.sh`](https://github.com/jaegertracing/jaeger/blob/main/plugin/storage/cassandra/schema/docker.sh#L21-L24):
```
if [ -z "$PASSWORD" ]; then
  ${CQLSH} ${CQLSH_SSL} ${CQLSH_HOST} ${CQLSH_PORT} -e "describe keyspaces"
else
  ${CQLSH} ${CQLSH_SSL} ${CQLSH_HOST} ${CQLSH_PORT} -u ${USER} -p ${PASSWORD} -e "describe keyspaces"
```

```
if [ -z "$PASSWORD" ]; then
  MODE="${MODE}" DATACENTER="${DATACENTER}" KEYSPACE="${KEYSPACE}" /cassandra-schema/create.sh "${TEMPLATE}" | ${CQLSH} ${CQLSH_SSL} ${CQLSH_HOST} ${CQLSH_PORT}
else
  MODE="${MODE}" DATACENTER="${DATACENTER}" KEYSPACE="${KEYSPACE}" /cassandra-schema/create.sh "${TEMPLATE}" | ${CQLSH} ${CQLSH_SSL} ${CQLSH_HOST} ${CQLSH_PORT} -u ${USER} -p ${PASSWORD}
fi
```

Is simply that `cqlsh -e "show version"` is so simple and straight forward that you don't think about a need of authentication to use it, so it's not until you run it from a remote machine when you realize of this (which indeed makes complete sense, to need auth for executing ANY command or query over a remote end).

Reply:
Also, the version check is only necessary if template is not provided.

Reply:
I think for the moment this would keep the current script behavior solving the problem and keeping consistency with the rest of the code, so I would accept it as a fix. Of course a proper refactor would be welcome if you guys find the time 🙂